src/main/java/uk/gov/hmcts/ccd/domain/service/search/CaseSearchResultViewGenerator.java

```diff
@@ -61,25 +64,78 @@ public CaseSearchResultView execute(String caseTypeId,
 
     private boolean itemsRequireFormatting(List<SearchResultViewHeaderGroup> headerGroups) {
         return headerGroups.size() == 1
-               && headerGroups.get(0).getFields().stream().anyMatch(field -> field.getDisplayContextParameter() != null);
+            && headerGroups.get(0).getFields().stream().anyMatch(field -> field.getDisplayContextParameter() != null);
     }
 
     private List<SearchResultViewItem> buildItems(String useCase, CaseSearchResult caseSearchResult, String caseTypeId, List<String> requestedFields) {
         CaseTypeDefinition caseTypeDefinition = getCaseTypeDefinition(caseTypeId);
         SearchResult searchResultDefinition = searchResultDefinitionService.getSearchResultDefinition(caseTypeDefinition, useCase, requestedFields);
```

Comment:
I know it's not part of your code changes but can we please rename `SearchResult` to `SearchResultDefinition`?

Reply:
Done 👍